### PR TITLE
[DEV-1390] Add Product schema and productsShowcase component to homepage in Strapi CMS

### PR DESCRIPTION
**Pull request by marcobottaro**
Add Product schema and productsShowcase component to homepage in Strapi CMS

#### List of Changes

- Add Product schema, controller, routes and service to Strapi CMS
- Add productsShowcase component to Strapi CMS

#### Screenshots (if appropriate):
Edit Homepage with the new section:
image

Products list:
image

Edit Product:
image

#### Types of changes

- [ ] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: apps/strapi-cms/src/api/homepage/content-types/homepage/schema.json [hunk 1]
```diff
@@ -4,17 +4,37 @@
...
+    "description": ""
   },
   "options": {
     "draftAndPublish": true
   },
-  "pluginOptions": {},
+  "pluginOptions": {
+    "i18n": {
+      "localized": true
+    }
```
Comment (kin0992) [inline apps/strapi-cms/src/api/homepage/content-types/homepage/schema.json:16]: why there is this change?

File: .changeset/sweet-planets-change.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"strapi-cms": patch
```
Comment (kin0992) [inline .changeset/sweet-planets-change.md:2]:
```suggestion
"strapi-cms": minor
```
IMHO this should be a minor

File: apps/strapi-cms/src/api/homepage/content-types/homepage/schema.json (hunk 1 above)
Comment (marcobottaro) [reply apps/strapi-cms/src/api/homepage/content-types/homepage/schema.json:16]: I was very hesitant, thanks!

File: .changeset/sweet-planets-change.md (hunk 2 above)
Comment (marcobottaro) [reply .changeset/sweet-planets-change.md]: I was very hesitant, thanks!

File: apps/strapi-cms/src/api/homepage/content-types/homepage/schema.json (hunk 1 above)
Comment (marcobottaro) [reply apps/strapi-cms/src/api/homepage/content-types/homepage/schema.json:16]: As the localization of the homepage was missing, the response to the GET `/api/homepage?populate%5BcomingsoonDocumentation%5D%5Bpopulate%5D%5B0%5D=links&populate%5BproductsShowcase%5D%5Bpopulate%5D%5B0%5D=products.logo` was not showing the localized `productsShowcase` and `comingsoonDocumentation` sections

File: apps/strapi-cms/src/api/product/content-types/product/schema.json [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    "description": {
+      "pluginOptions": {
+        "i18n": {
+          "localized": true
+        }
+      },
+      "type": "text"
+    },
+    "logo": {
+      "type": "media",
```
Comment (datalek) [inline apps/strapi-cms/src/api/product/content-types/product/schema.json:38]: Media content will be served through a CDN. We have two options to proceed: configure the plugin now ([here an example of the configuration](https://github.com/pagopa/b2b-portals/blob/cc58dd3/apps/strapi-cms/config/plugins.ts#L2-L19)) or add a task to the backlog to do it later. If we choose to add a task, we can insert the link to the task in this PR. In the task, we can also include the link to this PR

**Review by datalek — APPROVED** [review]

**Review by MarcoPonchia — APPROVED** [review]

**Pull request merged**